habitat-baselines/habitat_baselines/config/rearrange/rl_hierarchical.yaml

```diff
@@ -0,0 +1,66 @@
+# @package _global_
+
+defaults:
+  - /benchmark/rearrange: rearrange_easy
+  - /habitat_baselines: habitat_baselines_rl_config_base
+  - /habitat_baselines/rl/policy/obs_transforms:
+    - add_virtual_keys_base
+  - /habitat_baselines/rl/policy: hl_fixed
```

Comment:
Shouldn't this be hl_neural? It is strange that rl_hierarchical policy has a fixed high level policy.

Reply:
TP-SRL uses fixed HL policy but learned skills.

Reply:
Hm I think we were following a convention like 'tp_hierarchical' is the high level was fixed, and 'rl_hierarchical' if it was neural. If we don't support learning of low-level skills in the loop with the TP high-level, why do we call this 'rl' (there is no RL here right)?